syntax="proto2";

package grand;

option cc_generic_services = true;

message Snapshot {
    required int64 value = 1;
};

message IncrbyRequest {
    required int64 value = 1;
};

message GetRequest {
};

message LighterResponse {
    required bool success = 1;
    optional int64 value = 2;
    optional string redirect = 3;
};

service LighterService {
    rpc incrby(IncrbyRequest) returns (LighterResponse);
    rpc get(GetRequest) returns (LighterResponse);
};
